core/src/test/java/org/apache/james/core/MaybeSenderTest.java

```diff
@@ -155,4 +156,29 @@ void asStringWithDefaultShouldReturnDefaultWhenNullSender() {
             .isEqualTo("default");
     }
 
+
+
+    @Test
+    void getMailSenderShouldReturnNullSenderWhenNullSender() {
```

Comment:
missed when sender is `null` or `empty string`, `bad formated sender value` tests. could use parameterized test

Reply:
Indeed we can add the missing edge cases.

Concerning parametrized tests, it gets the syntax harder for little benefits. I'd rather not do it.